pkg/storaged/filesystem/mismounting.jsx

```diff
@@ -166,7 +202,10 @@
         }
 
         function do_unmount() {
-            return client.unmount_at(old_dir)
+            // HACK: old_dir is not defined if I have a subvolume mounted "rogue" in /foo and not in fstab, as
+            // old_dir comes from fstab and will be undefined.
+            // https://github.com/cockpit-project/cockpit/pull/19697
+            return client.unmount_at(old_dir || other)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-210454-49836fdb-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)